### PR TITLE
Add login page template

### PR DESCRIPTION
**Pull request by mirabela-t**
## Changelog

```changelog
 - add login page template
```

## Change Type

- [x] `Added`: new feature - non-breaking change which adds functionality.
- [ ] `Bug fix`: non-breaking change which fixes an issue.
- [ ] `Changed`: for changes in existing functionality.
- [ ] `Deprecated`: for soon-to-be removed features.
- [ ] `Documentation` update.
- [ ] `Performance`: for performance and optimization functionality.
- [ ] `Removed`: for now removed features.
- [ ] `Security`: in case of vulnerabilities.

## Required Checks

- [ ] Documentation
- [ ] Unit tests
- [x] Tested on Chrome
- [x] Tested on Firefox
- [x] Tested on IE11
- [x] npm run eslint
- [x] npm run prettier

## Screenshots

---

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import { BrowserRouter as Router } from 'react-router-dom'
 
+import './App.css'
```
Comment (utopiaeh) [inline src/index.js:5]: Move style file to last line

**Review by utopiaeh — APPROVED** [review]

File: src/pages/login/index.js [hunk 2]
```diff
@@ -1,7 +1,82 @@
...
+              />,
+            )}
+          </Form.Item>
+          <Form.Item>
+            <FormButton type="primary" htmlType="submit">
+              Continue
+              <Icon type="arrow-right" />
+            </FormButton>
+          </Form.Item>
+        </Form>
```
Comment (UncleFairy) [inline src/pages/login/index.js:68]: Now it is optimal solution, but in future , i assume, we will create a universal function form-creator due of big number of forms in our project.

**Review by UncleFairy — APPROVED** [review]

**Review by CapceleaCristian — APPROVED** [review]

**Review by andreiturcanu — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (mirabela-t) [reply src/index.js:5]: if I move it, the styles for the login button disappear for some reason

File: package.json [hunk 3]
```diff
@@ -10,6 +10,7 @@
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0",
+    "rxjs": "^6.5.3",
```
Comment (newalex997) [inline package.json]: is required for you at the moment to have `rxjs` installed?

**Review by newalex997 — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (mirabela-t) [reply package.json]: at the moment, no. I will uninstall it for now

**Review by utopiaeh — APPROVED** [review]

**Pull request merged**